src/widgets/scale/lv_scale.c

```diff
@@ -365,6 +369,10 @@ static void scale_draw_indicator(lv_obj_t * obj, lv_event_t * event)
     lv_draw_line_dsc_t line_dsc;
     lv_draw_line_dsc_init(&line_dsc);
     lv_obj_init_draw_line_dsc(obj, LV_PART_INDICATOR, &line_dsc);
+    /* Set tick line width to 2u when is not set */
+    if(0U == line_dsc.width) {
```

Comment:
I think it's not a good idea because there can be use cases when the main line is hidden.

Instead we should set the default line width from the theme. (I pushed a fix for it)

Reply:
Is this fix in master? I don't see it.

Reply:
Done in 4f000ae35, does it look ok?